src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/variable.py

```diff
@@ -16,6 +16,13 @@ def to_dto(self):
 
 
 class DefaultDatetime(datetime):
+    """
```

Comment:
We could move this class also

Reply:
Just wanted to start doing things right...not too much at the same time, but I agree :)